index.d.ts

```diff
@@ -663,8 +663,8 @@ export function collectDefaultMetrics(
 
 /**
  * Configure default metrics
- * @param timeout The interval how often the default metrics should be probed
- * @deprecated A number to defaultMetrics is deprecated, please use \`collectDefaultMetrics({ timeout: ${timeout} })\`.
+ * @param timeout Deprecated, ignored if present.
+ * @deprecated Deprecated, ignored if present.
```

Comment:
can we just remove the entire argument? next release is a major anyways

Reply:
we can do so (plus other deprecated stuff) separately, perhaps?

Reply:
how about do it now, it makes any subsequent changes easier if the deprecated code isn't there: https://github.com/siimon/prom-client/pull/330

Reply:
Sure, whatever's easiest!

Reply:
Hell yeah! I'm all for removing all deprecated stuffs.. been there for so long so it should be ok to remove those things now

Reply:
removed in #330 and I rebased this WIP on top of #330